src/routes/blog/content.ts

```diff
@@ -51,13 +51,15 @@ export const posts = Object.entries(postsGlob)
         return {
             title: frontmatter.title,
             description: frontmatter.description,
+            featured: frontmatter.featured,
             date: new Date(frontmatter.date),
             lastUpdated: new Date(frontmatter.lastUpdated),
             cover: frontmatter.cover,
             timeToRead: frontmatter.timeToRead,
             author: frontmatter.author,
             category: frontmatter.category,
-            href: `${base}/blog/post/${postName}`
+            href: `${base}/blog/post/${postName}`,
```

Comment:
lets add slug here too 👀 

that way we can use it as a proper index key for the each loop 👍🏻

Reply:
Done.